src/__tests__/unit/lib/boavizta/index.test.ts

```diff
@@ -100,6 +129,36 @@ describe('lib/boavizta: ', () => {
         expect(await output.execute([])).toEqual([]);
       });
 
+      it('throws an error when the metric type is `gpu-util`.', async () => {
```

Comment:
title says it's throws an error, but the implementation says it has result

Reply:
changed